src/components/molecules/AdminInput/AdminInput.tsx

```diff
@@ -41,7 +41,11 @@ export const AdminInput: React.FC<AdminInputProps> = ({
     "AdminInput--visible": !hidden,
   });
 
-  const hiddenClasses = classNames(parentClasses, "AdminInput__input");
+  const inputClasses = classNames(
+    "w-full shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md",
```

Comment:
Make the sausage as `TW.input` and just stick directly into `<input className={TW.input}`

In case of variants or other flag props, should probably shorten `classNames` as `cn` and just use `className={cn(TW.input, TW[variant])}`. Or put into variable, whichever allows for a more readable code.